docs/docs/50-user-guide/60-reference-docs/50-analysis-templates.md

```diff
@@ -97,6 +97,40 @@ spec:
         jsonPath: "{$.results.ok}"
 ```
 
+## Interval and Count
+
+To collect multiple measurements over a longer duration, use the 
+`count` and `interval` fields. This allows you to define how many
+measurements to take and how frequently to take them.
+
+In the example below, the analysis is configured to take 20
+measurements at 5-minute intervals, resulting in a total duration
+of approximately 1 hour:
```

Comment:
Wouldn't this be closer to two hours?

Other than that, lgtm

Reply:
🤦 oops. will fix it